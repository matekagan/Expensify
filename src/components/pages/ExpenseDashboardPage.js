import React from 'react';
import ExpensesList from '../presentational/ExpenseList';
import ExpenseListsFilters from '../presentational/ExpenseListFilters';
import ExpensesSummary from '../presentational/ExpenseSummary';

const ExpenseDashboardPage = () => (
    <div>
        <h3>Dashboard</h3>
        <ExpensesSummary />
        <ExpenseListsFilters />
        <ExpensesList />
    </div>
);

export default ExpenseDashboardPage;
