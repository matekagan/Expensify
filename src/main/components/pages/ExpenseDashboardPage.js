import React from 'react';
import ExpensesList from '../presentational/ExpenseList';
import ExpenseListsFilters from '../presentational/ExpenseListFilters';
import ExpensesSummary from '../presentational/ExpenseSummary';

const ExpenseDashboardPage = () => (
    <div>
        <ExpensesSummary />
        <ExpenseListsFilters />
        <ExpensesList />
    </div>
);

export default ExpenseDashboardPage;
