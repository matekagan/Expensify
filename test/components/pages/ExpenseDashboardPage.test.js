import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../../src/components/pages/ExpenseDashboardPage';

test('Should render expense dashboard page', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});
