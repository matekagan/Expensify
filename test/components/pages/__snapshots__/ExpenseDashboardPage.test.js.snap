// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render expense dashboard page 1`] = `
<div>
  <h3>
    Dashboard
  </h3>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
